hyco-https/tests/pipe.test.js

```diff
@@ -0,0 +1,138 @@
+var https = require('..');
+const Stream = require('stream');
+
+var ns = process.env.SB_HC_NAMESPACE ? process.env.SB_HC_NAMESPACE.replace(/^"(.*)"$/, '$1') : null;
+var path = process.env.SB_HC_PATH ? process.env.SB_HC_PATH : "a2";
+var keyrule = process.env.SB_HC_KEYRULE ? process.env.SB_HC_KEYRULE.replace(/^"(.*)"$/, '$1') : null;
+var key = process.env.SB_HC_KEY ? process.env.SB_HC_KEY.replace(/^"(.*)"$/, '$1') : null;
+
+expect(ns).toBeDefined();
+expect(path).toBeDefined();
+expect(keyrule).toBeDefined();
+expect(key).toBeDefined();
+
+var smallMessage = "SmallMessage";
+var kb = "";
+for (var i = 1024; i >= 0; i--) {
+    kb += String.fromCharCode(i % 128);
+}
+// let stream push 1kb at a time
+var over64kbMessage = []; 
+for (var j = 64; j >= 0; j--) {
```

Comment:
>j = 64; j >= 0 [](start = 9, length = 14)

Does this do 65? Is that intentional?

Reply:
was intentional to exceed 64kb

---
In reply to: [340271479](https://github.com/Azure/azure-relay-node/pull/44#discussion_r340271479) [](ancestors = 340271479)

Reply:
Ah, I should have noticed the name 'over64kbMessage'.

---
In reply to: [340271956](https://github.com/Azure/azure-relay-node/pull/44#discussion_r340271956) [](ancestors = 340271956,340271479)